# Hong Hu

_Bioinformatician_

[GitHub](https://github.com/vincenthhu/)

=====================

## Professional Summary

+ Bioinformatician with over seven years of hands-on working experience
+ Extensive expertise in NGS and multiomics data analysis/trouble-shooting
    - WGS, WES
    - Bulk/single-cell RNAseq
    - ChIPseq, ATACseq, BSseq, MEDIPseq, etc.
    - Analysis pipeline development and maintainance in HPC environment
+ Deep knowledege of Primary/Secondary analysis, Bioinformatics tools, databases
    - Variant calling, Peaking calling, Differential analysis, Pathway analysis, Annotation, QTL/GWAS, etc.
+ Proficiency in project consulting, planning, experiment design, result intepretation and grant/manuscript preparation
+ Strong programming skills with statistics and machine-learning background
    - R, Python, Shell, C++, nextflow, Anaconda, GitHub, etc.
    - Statistical modeling, svm, randomforest, Lasso, etc.
+ Strong communication skills, work ethics and fast-learning ability

## Working HISTORY




## Education

+ **Ph.D.--Bioinformatics**  
    *University of Illinois at Chicago* (2015)  
    **Thesis title**: Identification of transcriptional regulatory elements: novel machine learning approaches
    
## SELECTED PROJECTS

+ **WES data analysis of cancer patient samples**
    **Project Range**: WES analysis, including preprocessing, alignment, germline/somatic variant calling, contamination estimation and filtering, annotation
    
## PUBLICATIONS
+ Bao, L., Chau, C. S., Lei, Z., **Hu, H.**, Chan, A. G., Amber, K. T., Maienschein-Cline, M., & Tsoukas, M. M. (2021). Dysregulated microRNA expression in IL-4 transgenic mice, an animal model of atopic dermatitis. Archives of Dermatological Research, 313(10), 837–846. 
+ McGilvray, P. T., Anghel, S. A., Sundaram, A., Zhong, F., Trnka, M. J., Fuller, J. R., **Hu, H.**, Burlingame, A. L., & Keenan, R. J. (2020). An ER translocon for multi-pass membrane protein biogenesis. ELife, 9, e56889. 
+ Ebenezer, D. L., Fu, P., Krishnan, Y., Maienschein-Cline, M., **Hu, H.**, Jung, S., Madduri, R., Arbieva, Z., Harijith, A., & Natarajan, V. (2019). Genetic deletion of Sphk2 confers protection against Pseudomonas aeruginosa mediated differential expression of genes related to virulent infection and inflammation in mouse lung. BMC Genomics, 20, 984. 
+ Xuan, W., Wang, Y., Tang, Y., Ali, A., **Hu, H.**, Maienschein-Cline, M., & Ashraf, M. (2018). Cardiac Progenitors Induced from Human Induced Pluripotent Stem Cells with Cardiogenic Small Molecule Effectively Regenerate Infarcted Hearts and Attenuate Fibrosis. Shock (Augusta, Ga.), 50(6), 627–639. 
+ Hu, W., Hu, D., Xie, L., **Hu, H.**, Li, Y., Nonn, L., Shioda, T., & Prins, G. S. (2018). Abstract 3063: Stem cell lineage hierarchy by keratin profiling in normal human prostate epithelial cells and prostate cancer. Cancer Research, 78(13 Supplement), 3063–3063. 
+ Ebenezer, D. L., Fu, P., Krishnan, Y. H., Jung, S. C., **Hu, H.**, Arbieva, Z., Madduri, R., Harijith, A. K., & Natarajan, V. (2018). Expression profiling of genes regulated by Sphingosine kinase 2 in a murine model of Pseudomonas aeruginosa mediated acute lung inflammation. The FASEB Journal, 32(S1), 540.11-540.11. 
+ Cook, L. C. C., **Hu, H.**, Maienschein-Cline, M., & Federle, M. J. (2018). A Vaginal Tract Signal Detected by the Group B Streptococcus SaeRS System Elicits Transcriptomic Changes and Enhances Murine Colonization. Infection and Immunity, 86(4), e00762-17
+ Maker, A., Green, S., **Hu, H.**, Brugge, W., Winter, J., Hackert, T., Buchler, M., Lawler, R., Scarpa, A., & Bassi, C. (2017). A single-platform cyst fluid assay to predict IPMN with high-malignant potential for surgical resection: A multi-institutional international report. HPB, 19, S7–S8. 
+ Hu, W.-Y., Hu, D.-P., Xie, L., Li, Y., Majumdar, S., Nonn, L., **Hu, H.**, Shioda, T., & Prins, G. S. (2017). Isolation and Functional Interrogation of Adult Human Prostate Epithelial Stem Cells at Single Cell Resolution. Stem Cell Research, 23, 1–12
+ Xu, J., **Hu, H.**, & Dai, Y. (2016). LMethyR-SVM: Predict Human Enhancers Using Low Methylated Regions based on Weighted Support Vector Machines. PloS One, 11(9), e0163491. 
+ **Hu, H.**, Xu, J., & Dai, Y. (2015). Regulatory elements in low-methylated regions predict directional change of gene expression. IEEE Journal of Biomedical and Health Informatics, 19(4), 1293–1300. 
+ **Hu, H.**, & Dai, Y. (2014). Exploring regulatory elements in low-methylated regions for gene expression prediction. 2014 36th Annual International Conference of the IEEE Engineering in Medicine and Biology Society, 4763–4766.
+ **Hu, H.**, & Dai, Y. (2014). A model-based approach to transcription regulatory network reconstruction from time-course gene expression data. Annual International Conference of the IEEE Engineering in Medicine and Biology Society. IEEE Engineering in Medicine and Biology Society. Annual International Conference, 2014, 4767–4770.
+ Mehta, S. D., Green, S. J., Maclean, I., **Hu, H.**, Bailey, R. C., Gillevet, P. M., & Spear, G. T. (2012). Microbial Diversity of Genital Ulcer Disease in Men Enrolled in a Randomized Trial of Male Circumcision in Kisumu, Kenya. PLoS ONE, 7(7), e38991.
+ **Hu, H.**, Roqueiro, D., & Dai, Y. (2011). Prioritizing predicted cis-regulatory elements for co-expressed gene sets based on Lasso regression models. 2011 Annual International Conference of the IEEE Engineering in Medicine and Biology Society, 6853–6856.
+ Book chapter: **Hu, H.**, & Dai, Y. (2016). Prioritize Transcription Factor Binding Sites for Multiple Co-Expressed Gene Sets Based on Lasso Multinomial Regression Models. In Emerging Research in the Analysis and Modeling of Gene Regulatory Networks (pp. 280–315). IGI Global

## REVIEW BOARD

+ IEEE Journal of Biomedical and Health Informatics. 2017
+ Book chapter in Emerging Research in the Analysis and Modeling of Gene Regulatory Networks. IGI Global. 2015
+ Annual International Conference of the IEEE Engineering in Medicine and Biology Society, 2014
+ Clinical, Cosmetic and Investigational Dermatology.  2013

You can use the [editor on GitHub](https://github.com/vincenthhu/digital-cv/edit/gh-pages/index.md) to maintain and preview the content for your website in Markdown files.

Whenever you commit to this repository, GitHub Pages will run [Jekyll](https://jekyllrb.com/) to rebuild the pages in your site, from the content in your Markdown files.

### Markdown

Markdown is a lightweight and easy-to-use syntax for styling your writing. It includes conventions for

```markdown
Syntax highlighted code block

# Header 1
## Header 2
### Header 3

- Bulleted
- List

1. Numbered
2. List

**Bold** and _Italic_ and `Code` text

[Link](url) and ![Image](src)
```

For more details see [Basic writing and formatting syntax](https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax).

### Jekyll Themes

Your Pages site will use the layout and styles from the Jekyll theme you have selected in your [repository settings](https://github.com/vincenthhu/digital-cv/settings/pages). The name of this theme is saved in the Jekyll `_config.yml` configuration file.

### Support or Contact

Having trouble with Pages? Check out our [documentation](https://docs.github.com/categories/github-pages-basics/) or [contact support](https://support.github.com/contact) and we’ll help you sort it out.
